src/utils/router.js

```diff
@@ -109,45 +102,26 @@ export const computeTag = ({ to, disabled, routerComponentName } = {}, thisOrPar
   return routerComponentName || (thisOrParent.$nuxt ? 'nuxt-link' : 'router-link')
 }
 
-export const computeRel = ({ target, rel } = {}) => {
-  if (target === '_blank' && isNull(rel)) {
-    return 'noopener'
-  }
-  return rel || null
-}
-
-export const computeHref = (
-  { href, to } = {},
-  tag = ANCHOR_TAG,
-  fallback = '#',
-  toFallback = '/'
-) => {
-  // We've already checked the $router in computeTag(), so isRouterLink() indicates a live router.
-  // When deferring to Vue Router's router-link, don't use the href attribute at all.
-  // We return null, and then remove href from the attributes passed to router-link
-  if (isRouterLink(tag)) {
```

Comment:
@Hiws I've looked up when and why this was added and could track it down to this PR way back in 2017:
https://github.com/bootstrap-vue/bootstrap-vue/issues/1015

I've created a simple demo on CodeSandbox to make sure this still works:
https://codesandbox.io/s/bootstrapvue-starter-project-forked-dihox